staging/src/k8s.io/apiserver/pkg/util/flowcontrol/config.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	fcrequest "k8s.io/apiserver/pkg/util/flowcontrol/request"
+	"sigs.k8s.io/yaml"
+)
+
+// PriorityAndFairnessConfig holds priority and fairness parameters.
+type PriorityAndFairnessConfig struct {
+	// WorkEstimatorConfig holds work estimator parameter values.
+	*fcrequest.WorkEstimatorConfig `json:"workEstimatorConfig,omitempty"`
+}
+
+// NewConfig creates a new PriorityAndFairnessConfig with default values.
+func NewConfig() PriorityAndFairnessConfig {
+	return PriorityAndFairnessConfig{
+		WorkEstimatorConfig: fcrequest.DefaultWorkEstimatorConfig(),
+	}
+}
+
+// ReadPriorityAndFairnessConfig parses configuration stored under configFilePath and overrides
+// PriorityAndFairnessConfig cfg object with defined values (leaving undefined ones unchanged).
+func ReadPriorityAndFairnessConfig(configFilePath string, cfg *PriorityAndFairnessConfig) error {
```

Comment:
In the spirit of the above comment - maybe just call it: ReadConfigFromFile?

The package determines what kind of config we're talking about.

Reply:
Done.